frontend/templates/product/sections/ProductReviewsSection/index.tsx

```diff
@@ -217,15 +219,41 @@ function ReviewsStats({
 
 type ProductReviewLineItemProps = {
    review: ProductReview;
+   variantSku: string | undefined | null;
 };
-function ProductReviewLineItem({ review }: ProductReviewLineItemProps) {
+function ProductReviewLineItem({
+   review,
+   variantSku,
+}: ProductReviewLineItemProps) {
+   const isAdminUser = useAuthenticatedUser().data?.isAdmin ?? false;
+   const appContext = useAppContext();
+
    return (
       <Box
          py="6"
          borderBottomWidth="1px"
          borderColor="gray.200"
          data-testid="product-review-line-item"
       >
+         {isAdminUser && (
+            <Button
+               variant="link"
+               as={Link}
+               bgColor="transparent"
+               textColor="brand"
+               fontFamily="heading"
+               lineHeight="1.29"
```

Comment:
This is a very specific line height. I'm curious how you came up with it

Reply:
I just copied the Edit button stuff from here https://github.com/iFixit/react-commerce/blob/4f9dcdda5c19800960ae9d96827a5fb83fa27dca/frontend/templates/troubleshooting/index.tsx#L204

I did this to keep the design of that button the same everywhere. That's where these values come from.

Reply:
Ah I see, thanks for explaining!